### PR TITLE
[wrangler] types command

### PR DESCRIPTION
**Pull request by JacobMGEvans**
Add documentation reference in command section of Wrangler for new command feature.

### CODE REVIEW

**Review by Nuttapon43 — COMMENTED** [review]
@JacobMGEvans

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Wrangler offers a number of commands to manage your Cloudflare Workers.
 - [`login`](#login) - Authorize Wrangler with your Cloudflare account using OAuth.
 - [`logout`](#logout) - Remove Wrangler’s authorization for accessing your account.
 - [`whoami`](#whoami) - Retrieve your user information and test your authentication configuration.
+- [`types`](#types) - Generate types from bindings & module rules in configuration.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
- [`types`](#types) - Generate types from bindings and module rules in configuration.
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -1185,3 +1186,13 @@ Retrieve your user information and test your authentication configuration.
 ```sh
 $ wrangler whoami
 ```
+
+---
+
+## types
+
+Generate types from bindings & module rules in configuration.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
Generate types from bindings and module rules in configuration.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX review with suggestions

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (penalosa) [reply content/workers/wrangler/commands.md]: Would it be possible to add more detail here? Maybe an example of a TOML file, and how it gets translated (and the name & location of the generated file)

**Pull request merged**